src/dialog_launcher.ts

```diff
@@ -141,6 +141,33 @@ export class Launcher extends search.Search {
             // Truncate excess items from the list
             this.options.splice(this.list_max());
 
+            // Fall back to web query if nothing found
```

Comment:
Maybe we should have a search-not-found condition for plugins?

Reply:
Hmmm... I wonder what a good name for a setting would be, kind of the "Default Prefix if no results found".

Reply:
I've removed this for now since it's a bigger behavior change.